storage/s3_sdk/list_gcs_buckets.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+// Copyright 2019 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START storage_s3_sdk_list_buckets]
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+func list_gcs_buckets(googleAccessKeyID string, googleAccessKeySecret string) ([]*s3.Bucket, error) {
```

Comment:
Please pass an `io.Writer` as an argument. See #650 for an example for what it should look like and how to test it.